clang/lib/Sema/SemaChecking.cpp

```diff
@@ -168,7 +168,7 @@ bool Sema::checkArgCount(CallExpr *Call, unsigned DesiredArgCount) {
 
```

Comment:
Maybe also assert that `ArgCount > 0`? In case `checkArgCountAtLeast` becomes broken.

Reply:
I think that's covered by:
```C++
  assert(ArgCount > DesiredArgCount && "should have diagnosed this");
```
since `DesiredArgCount` is nonnegative.